hadoop-hdds/rocksdb-checkpoint-differ/src/main/java/org/apache/ozone/rocksdiff/RocksDBCheckpointDiffer.java

```diff
@@ -455,9 +447,6 @@ public void onCompactionBegin(RocksDB db,
           return;
         }
```

Comment:
CompactionJobInfo has function to get the column family. You can skip the whole compaction listener here. No need for checking individual files. https://github.com/facebook/rocksdb/blob/dfaf4dc111ff090800f2765fdcd88d48a6abbab8/java/src/main/java/org/rocksdb/CompactionJobInfo.java#L32

```suggestion
        }
        if (!COLUMN_FAMILIES_TO_TRACK_IN_DAG.contains(StringUtils.bytes2String(compactionJobInfo.columnFamilyName())) {
        return;
   }
```

Reply:
Combine both the if statements. Unable to make the suggestion there. In this case the other changes need not be done. Add a unit test for this function alone.